pkgs/applications/file-managers/vifm/default.nix

```diff
@@ -17,9 +18,14 @@ in stdenv.mkDerivation rec {
     sha256 = "sha256-j+KBPr3Mz+ma7OArBdYqIJkVJdRrDM+67Dr2FMZlVog=";
   };
 
-  nativeBuildInputs = [ pkg-config makeWrapper ];
+  nativeBuildInputs = [ perl pkg-config makeWrapper ];
   buildInputs = [ ncurses libX11 util-linux file which groff ];
```

Comment:
Wouldn't this need to go to buildInputs for patchshebang?

Reply:
If we consider a cross-compilation case (`--build=x86_64-linux` `--host=riscv64-linux`). There can be two uses of `perl` in the derivation:
- use at build time: needs to run on `x86_64-linux`, exposed via `nativeBuildInputs` (or `depsBuild*` for cross-compilers themselves)
- use at run time: needs to run on `riscv64-linux`, exposed via `buildInputs` (or `depsHost*` for cross-compilers themselves)

`src/helpztags` is used at build time. Thus `perl` is pulled into `nativeBuidInputs`. Note that `patchShebangs` also uses `--build` to use build-time `perl`.

It probably does not matter much as `vifm` does not cross-compile as is anyway and files as:

```
$ nix build --no-link nixpkgs#pkgsCross.riscv64.vifm
...
  GEN      ../data/vifm-help.txt
 /<<NIX>>/coreutils-9.1/bin/mkdir -p '/<<NIX>>/vifm-riscv64-unknown-linux-gnu-0.12.1/share/vifm'
 /<<NIX>>/coreutils-9.1/bin/install -c -m 644 ./../data/vifm-help.txt '/<<NIX>>/vifm-riscv64-unknown-linux-gnu-0.12.1/share/vifm'
/<<NIX>>/coreutils-9.1/bin/install: cannot stat './../data/vifm-help.txt': No such file or directory
```